src/pyJiraCli/jira_server.py

```diff
@@ -56,11 +57,20 @@ class Server:
     """
     def __init__(self):
         self._crypto_h = Crypto()
+        self._error_h = Error()
```

Comment:
Does it matter using always the same instance? If not, instantiate it only in case you need it.